ompi/debuggers/predefined_gap_test.c

```diff
@@ -71,6 +71,7 @@ int main(int argc, char **argv) {
 #else
     GAP_CHECK("error_handler", test_comm, error_handler, c_f_to_c_index, 1);
 #endif
+    GAP_CHECK("errhandler_type", test_comm, errhandler_type, error_handler, 1);
```

Comment:
I think there is another line that needs to be added back (in fact, looking at it now I am not sure the code was correct before, but I am not completely sure I understand how the debugger interfaces work). It seems that we are always checking for the distance from the previous element of the structure.
`GAP_CHECK("c_pml_comm", test_comm, c_pml_comm, errhandler_type, 1);`

Reply:
Thank you!

added in new revision